Wiki/Modules/FvwmRearrange/index.md

```diff
@@ -27,4 +27,60 @@ current screen subject to certain constraints. Layering is performed
 so consecutive windows will have their window titles visible underneath
 the previous.
 
+## Configuration and Use
+
+FvwmRearrange is invoked from a menu, pop-up, or button. Several
+command line options can be used to constrain the layering. For a full
+list of FvwmRearrange configuration options see the FvwmRearrange manpage.
+
+Here is a sample configuration for tiling with 2 columns.
+
+{% highlight fvwm %}
+FvwmRearrange -tile $0 -r -mn 2 -maximize 0 0 $[wa.width]p $[wa.height]p
+{% endhighlight %}
+
+To restore the windows, FvwmRearrange doesn't reverse the options,
+use this for all.
+
+{% highlight fvwm %}
+All (CurrentPage, !Iconic, CirculateHit, !Sticky) Maximize Off
+WindowId $0 WarpToWindow 50 50
+{% endhighlight %}
+
+## Example
+
+This advanced example is attached to key binding (alt-t) with two functions
+that do the tiling with 3 columns. With the same key binding, it restores all
+windows. This configure does like the following to the windows, before and
+after tiling :
+
+|![image](before-after-tile.png)|
+
+Tiling with 3 columns.
+
+{% highlight fvwm %}
+DestroyFunc 3Tile
+AddToFunc 3Tile
++ I FvwmRearrange -tile $0 -r -mn 3 -maximize 0 0 86 82 $[wa.width]p $[wa.height]p
+{% endhighlight %}
+
+Tile switch on and off.
+
+{% highlight fvwm %}
+SetEnv TileSwitch ON
+
+DestroyFunc Tile
+AddToFunc Tile
++ I Piperead 'if [ $TileSwitch = ON ]; then echo "3Tile"; \
+	echo "Schedule 500 SetEnv TileSwitch OFF"; fi'
++ I Piperead 'if [ $TileSwitch = OFF ]; \
+	then echo "All (CurrentPage, !Iconic, CirculateHit, !Sticky) Maximize Off"; \
+	echo "WindowId $0 WarpToWindow 50 50"; echo "SetEnv TileSwitch ON"; fi'
```

Comment:
So I suspect the reason you're using `Schedule` here is because of the successive `PipeRead` commands being one after the other.

This is undesirable -- and instead you should probably toggle outside of those PipeRead commands.

Bonus points for using InfoStoreAdd here as well.